HelloWorld.php

```diff
@@ -9,7 +9,7 @@ class HelloWorld extends Controller {
   protected $greeting;
 
   public function __construct(string $greeting = '') {
-    $this->greeting = $greeting.
+    $this->greeting = $greeting;
```

Comment:
While this is technically correct, I'd suggest using the already available `setGreeting(string  $greeting): void` method so that any validation added at later periods can be already applied when the object is instantiated.